### DISCUSSION TITLE
RadioGroup focus support

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2556] [Q&A]
Does RadioGroup support focus? Because i can not trigger it

**Comment by SevenOutman** [comment]
Could you provide a CodeSandbox demo? You may use this template https://codesandbox.io/s/rsuite-template-5vq6zo2z5l

**Reply by abdullaev388** [reply]
Here is it:
https://codesandbox.io/s/musing-torvalds-zjbxsh

onClick is working but onFocus is not.
